src/models/fetch_org_response.rs

```diff
@@ -37,6 +37,13 @@ pub struct FetchOrgResponse {
     pub custom_role_mapping_name: Option<String>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub legacy_org_id: Option<String>,
+    pub url_safe_org_slug: String,
+    pub can_setup_saml: bool,
+    pub is_saml_in_test_mode: bool,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub domain: Option<String>,
+    pub domain_autojoin: bool,
+    pub domain_restrict: bool,
 }
```

Comment:
So unfortunately, this type is also used under `FetchOrgsResponse` in the `orgs` vec, and the response data there doesn't provide any of these new fields currently. To handle this we'd need `FetchOrgsResponse` to look like
```
FetchOrgsResponse {
    #[serde(rename = "orgs")]
    pub orgs: Vec<crate::models::FetchOrgBasicResponse>,
    ...
}
```
where `FetchOrgBasicResponse` looks how `FetchOrgResponse` does currently.